scripts/install-voice.sh

```
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Install local voice pipeline: faster-whisper (STT) + Kokoro (TTS)
+# Runs as clawbox user. Requires espeak-ng to be installed (system package).
+set -euo pipefail
+
+CLAWBOX_USER="clawbox"
+CLAWBOX_HOME="/home/clawbox"
```

Shellcheck (0.11.0)

[warning] CLAWBOX_HOME appears unused. Verify use (or export if used externally).

(SC2034)